File: python/code/119_pattern11.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(n-1,i,-1):
        print(" ", end='')
    for k in range(2*(i+1)):
        if(k == 0):
            print("/", end='')
        elif(k == 2*(i+1)-1):
            print("\\", end='')
        elif(n//2 == i):
            print("_", end='')
        else:
            print(" ", end='')
    print()
    
# Output :-
# Enter n :- 5
#     /\
#    /  \
#   /____\
#  /      \
# /        \
 

File: c/code/119_pattern_11.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=n-1;j>i;j--) {
            printf(" ");
        }
        for(int k=0;k<2*(i+1);k++) {
            if(k == 0) printf("/");
            else if(k == 2*(i+1)-1) printf("\\");
            else if(n/2 == i) printf("_");
            else printf(" ");
        }
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
    /\
   /  \
  /____\
 /      \
/        \

*/